doc/USAGE.md

```diff
@@ -73,6 +73,17 @@ Take for example the following route modification command - `sudo route add -net
 ```
 192.168.1.0/24 via 192.168.0.1 dev em1
 ```
+Alternatively, if you run 'ip route get 192.168.1.0' on the host, a correct output would look like:
+```
+192.168.1.0 via 192.168.0.1 dev eno1  src 192.168.0.11
+    cache

```

Comment:
It means that all packets with a destination address of 192.168.1.0 will be routed via 192.168.0.1 after adding the routes using network config script

Reply:
I think it means there is not cache entry for this route. In case a cache entry existed, the output would look something like this:
192.168.100.17 via 192.168.99.254 dev eth0  src 192.168.99.35 
    cache  users 1 used 326 age 12sec mtu 1500 advmss 1460

Reply:
Done